packages/beacon-node/src/network/reqresp/types.ts

```diff
@@ -52,6 +61,10 @@ export const isSingleResponseChunkByMethod: {[K in Method]: boolean} = {
   [Method.Metadata]: true,
   [Method.BeaconBlocksByRange]: false, // A stream, 0 or more response chunks
   [Method.BeaconBlocksByRoot]: false,
+  [Method.LightClientBootstrap]: true, // TODO DA: confirm
+  [Method.LightClientUpdate]: false, // TODO DA: confirm
+  [Method.LightClientFinalityUpdate]: true, // TODO DA: confirm
+  [Method.LightClientOptimisticUpdate]: true, // TODO DA: confirm
```

Comment:
looks good, everything but the LightClientUpdate protocol should be a single response chunk